terraform/modules/govuk/variables.tf

```diff
@@ -40,6 +40,11 @@ variable "documentdb_security_group_id" {
   type        = string
 }
 
+variable "mongodb_security_group_id" {
```

Comment:
Is there just one mongodb instance? Just checking if this name is future proof

Reply:
there is only one, the other one is called `router_backend`, at least in gov.uk speak

Reply:
Pretty sure it is, since the name is govuk_mongo_access.

Reply:
This one's the general/shared mongodb so I think it's probably ok. We could rename it to shared_mongodb (I think we used that pattern for shared_documentdb so would make sense.)